### PR TITLE
add read-only property

### PR DESCRIPTION
**Pull request by omsmith**

### CODE REVIEW

File: d2l-alignment-list.html [hunk 1]
```diff
@@ -160,7 +162,7 @@
 			},
 
 			_hasStartUpdateAlignments: function(entity) {
```
Comment (jstefaniuk-d2l) [inline d2l-alignment-list.html]: should this be `_hasStartUpdateAlignments: function(entity, readonly) {` ?

File: d2l-alignment.html [hunk 2]
```diff
@@ -45,15 +45,16 @@
 
 		<div>
 			<d2l-alignment-intent href="[[_getIntent(entity)]]" token="[[token]]"></d2l-alignment-intent>
-			<template is="dom-if" if="[[_hasRemoveAlignment(entity)]]">
+			<template is="dom-if" if="[[!readOnly && _hasRemoveAlignment(entity)]]">
```
Comment (jstefaniuk-d2l) [inline d2l-alignment.html]: I don't think you can `&&` in a binding without breaking this into it's own function (https://www.polymer-project.org/1.0/docs/devguide/data-binding)

File: d2l-alignment.html [hunk 3]
```diff
@@ -45,15 +45,16 @@
...
 				<d2l-button-icon icon="d2l-tier1:close-default" text="[[localize('removeAlignment')]]" on-click="_remove"></d2l-button-icon>
 			</template>
 		</div>
 		<template is="dom-if" if="[[_hasDemonstrations(entity)]]">
 			<d2l-outcomes-level-of-achievements
 				token="[[token]]"
-				href="[[_getDemonstrations(entity)]]">
-			</d2l-outcomes-level-of-achievements>
+				href="[[_getDemonstrations(entity)]]"
+				read-only="[[readOnly]]"
```
Comment (jstefaniuk-d2l) [inline d2l-alignment.html]: does this file need a properties object containing "readOnly"?

Comment (omsmith) [reply d2l-alignment.html]: Thought I did, oops

**Pull request merged**